src/platform_impl/windows/event_loop.rs

```diff
@@ -638,13 +632,6 @@ unsafe extern "system" fn public_window_callback<T: 'static>(
         }
 
         winuser::WM_NCLBUTTONDOWN => {
-            // jumpstart the modal loop
```

Comment:
This `RedrawWindow` and the `PostMessageW` below it were to prevent the main loop from pausing when the user clicks the title bar.